lpsgateway/lpsgateway.ino

```diff
@@ -27,44 +55,35 @@ void loop()
     delay(10);
     rf95.send(&radiopacket, 1);
     rf95.waitPacketSent();
-
     delay(10);
-    if (rf95.waitAvailableTimeout(1000))
-    {  
-      uint8_t buf[RH_RF95_MAX_MESSAGE_LEN];
-      uint8_t len = sizeof(buf);
-      if (rf95.recv(buf, &len))
-      {
-        Serial.write(*buf);Serial.write('\r');Serial.write('\n');
-      }
+  }
+  if (rf95.waitAvailableTimeout(100))
+  {  
+    uint8_t buf[RH_RF95_MAX_MESSAGE_LEN];
+    uint8_t len = sizeof(buf);
+    if (rf95.recv(buf, &len))
+    {
+      Serial.write(buf, len);
     }
   }
 }
 
 void init_communication()
-{ // Initialize the communication link
-
-  /*
-   * Serial link
-   */
-  Serial.begin(BAUDRATE);
```

Comment:
Why not use 'baudrate' thats equal to 115600?

Reply:
That's a mistake